client/src/components/Navlinks/style.css

```diff
@@ -0,0 +1,70 @@
+
+button {
+    -webkit-transition: background-color .3s;
+    transition: background-color .3s;
+    font-size: 1rem;
+    display: block;
+    padding: 0 15px;
+    cursor: pointer;
+    border: none;
+    height: 64px;
+    margin: 0;
+    text-decoration: none;
+    color: #fff;
+    font-family: sans-serif;
+    font-size: 1rem;
+    cursor: pointer;
+    text-align: center;
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    background-color: #9b74c2;
```

Comment:
there is that hex color that i have to figure out again by looking it up. make this a global css variable